### ISSUE TITLE
Polymer 2 support

### ISSUE THREAD

**Issue by masonlouchart** [saeidzebardast/moment-js#28]
_It will be great if there was a good samaritan to migrate this element to [Polymer 2](https://www.polymer-project.org/2.0/docs/upgrade)..._ 🤔

**Comment by masonlouchart** [comment]
I'm working on it but I found our unit tests suites are inefficient. They assert on _"moment-is-ready"_ event without "calling properly `done`" at the end, so they all pass without being executed.

Only the test [at line 50](https://github.com/saeidzebardast/moment-js/blob/master/test/basic-test.html#L50) is good because it defines the `done` parameter.

To check what I mean it's simple. Add `console.log` after assertions and run tests with option `-p`.
Run `polymer test -l chrome -p` for example. Open the console and see there is no logs.

By adding the `done` parameter (like at the line 50) they all fail because of timeout (excepted the first one but it is the only good and was already passing).

### To summarize:
- I have an hybrid element (working with Polymer 1 and 2)
- I try to fix unit tests that are actually useless (for now)

**Comment by masonlouchart** [comment]
I opened an other issue related to useless unit test suite. This one will be closed when my PR will be merged.
